content/calico/stars_policy_demo/create_resources.md

```diff
@@ -15,7 +15,13 @@ cd ~/environment/calico_resources
 
 #### Stars Namespace
 
-Save the following as `namespace.yaml`.
+Copy/Paste the following commands into your Cloud9 Terminal.
+```
+cd ~/environment/calico_resources
```

Comment:
You can remove this command (seems redundant from previous command)
cd ~/environment/calico_resources

Reply:
I'm leaving it in there, because it's harmless, and I want to make sure we get the file in the right place.